templates/mongodb-statefulset.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1beta2
```

Comment:
I'd preffer to use mongodb as requirenment dependency `stable/modngodb` with values from this chart